src/main/java/no/nav/pto/veilarbportefolje/util/DateUtils.java

```diff
@@ -176,4 +178,15 @@ public static String nowToStr() {
     public static ZonedDateTime now() {
         return ZonedDateTime.now().truncatedTo(ChronoUnit.MICROS);
     }
-}
+
+    public static Comparator<LocalDate> closestToTodayComparator() {
+        long todayEpoch = fromLocalDateToEpochSecond(LocalDate.now());
+        return Comparator.comparing(dato -> abs(fromLocalDateToEpochSecond(dato)-todayEpoch));
+    }
+
+    private static long fromLocalDateToEpochSecond(LocalDate date){
+        return date.toEpochSecond(date.atStartOfDay().toLocalTime(), ZoneOffset.UTC);
+    }
+
```

Comment:
For å bruke Epochtid må man ha tid og tidssone. Jeg prøvde egentlig å bare brukte dato uten tid men fant ingen god måte å sammenligne datoer. Kan godt hende det finnes men jeg endte altså med å bruke epoch. Da må jeg jo legge til en "fake" tid og tidssone, men siden alle får samme så påvirker det jo ikke forholdet mellom dem. Men mulig noen har noen innvendiger her